.gitignore

```diff
@@ -36,5 +36,6 @@ tests-ui/workflows/examples
 /playwright/.cache/
 
 .env
+.vscode/settings.json
```

Comment:
There's already a `.vscode/*` above which covers that

Reply:
🙏